src/components/AssetInput/AssetInput.tsx

```diff
@@ -82,9 +84,14 @@ export type AssetInputProps =
           size?: undefined;
           onUploadClick?: () => void;
           onLibraryClick?: () => void;
+          isLoading?: boolean;
       };
 
-const AssetThumbnail: FC<Pick<AssetProps, "asset" | "size"> & { isActive?: boolean }> = ({ asset, size, isActive }) => (
+const AssetThumbnail: FC<Required<Pick<AssetProps, "asset" | "size">> & { isActive?: boolean }> = ({
```

Comment:
The type is becoming more complex => make it simpler and extract to own type?
```ts
type AssetThumbnailProps = {
  asset: Required<AssetProps['asset']>,
  size: AssetProps['size'],
  isActive?: boolean
}
```

Reply:
Done 👍